tools/osv-generator/generator.go

```diff
@@ -26,9 +26,8 @@ import (
 const URL = "https://security.access.redhat.com/data/csaf/v2/advisories"
 
 // Get list of new advisories published in the last `days`
-// TODO: Do not use until published.csv is periodically generated
 func getAdvisoryListByPublished(days int) ([]string, error) {
```

Comment:
Since we're using `releases.csv` instead of `published.csv`, how about renaming this to `getAdvisoryListByReleases`?